css/style.css

```diff
@@ -0,0 +1,123 @@
+html, body{
+    background: #3f3d3d;
+}
+
+.wrapper{
+    width: 90%;
+    max-width: 900px;
+    margin: 30px auto;
+    background: #f3f3f3;
+}
+
+header{
+    padding: 25px 35px;
+    border-bottom: 2px solid #2c78ba;
+    height: 150px;
+}
+
+h1{
+    font-size: 30px;
+	font-weight: 700;
+	font-family: "Lato", Arial, sans-serif;
+	margin-bottom: 10px;
+}
+
+h2{
+    font-size: 25px;
+	margin-left: 2px;
+	font-family: "Lato", Arial, sans-serif;
+}
+
+main{
+    padding: 0 40px;
+}
+
+.profile-photo{
+    width: 20%;
+	float: left;
+	margin-right: 25px;
+}
+
+.profile-photo img{
+    width: 100px;
+    height: 100px;
+    border-radius: 75px;
+    box-shadow: 0 0 20px #2c78ba; 
+}
+
+.name{
+    float: left;
+}
+
+.contact-details{
+    float: right;
+}
+
+.contact-details ul{
+    list-style-type: none;
+    font-family: "Lato", Arial, sans-serif;
+	font-size: 13px;
+	margin-top: 2px;
+}
+
+.contact-details ul li{
+    margin-bottom: 4px;
+    color: #545759;
+}
+
+.contact-details ul li a:hover{
+    color: #2c78ba;
+}
+
+section{
+    border-top: 1px solid #2c78ba ;
+    padding: 20px 0 0;
+    height: 170px;
```

Comment:
forgot make clearfix

Reply:
and better to make section > div like inline-block element / use flexbox to avoid right and left floating